projects/core/src/product/model/product-scope.ts

```diff
@@ -14,5 +14,5 @@ export const enum ProductScope {
   STOCK = 'stock',
   PROMOTIONS = 'promotions',
   LIST_ITEM = 'list_item',
-  MULTIDIMENSIONAL = 'multidimensional',
+  MULTI_DIMENSIONAL = 'multi_dimensional',
```

Comment:
`nitpick`: I've checked and both versions, `multidimensional` and `multi-dimensional` are correct. It would be good to ask someone (maybe Bill) what version we want to use in official docs and adjust the code to be in line with documentation and how this feature is going to be introduced to the customers.

Reply:
I've already decided to go with `Multi-Dimensional`, here is past conversation https://github.com/SAP/spartacus/pull/19040#discussion_r1682603065